client/js/lounge.js

```diff
@@ -391,6 +389,15 @@ $(function() {
 	}
 
 	function renderNetworks(data) {
+		data.networks.forEach(function(network) {
```

Comment:
Server *guarantees* the first channel is a lobby. probably simpler to just pop it from the list.